# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../maverik/compile-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur maverik-6.2-orig/setup maverik-6.2/setup
--- maverik-6.2-orig/setup	2002-03-20 17:59:51.000000000 +0000
+++ maverik-6.2/setup	2006-01-26 19:50:36.854952250 +0000
@@ -6,7 +6,7 @@
 TDMPATH=""
 MESAPATH=""
 MESADEFAULT=1
-XLIBPATH="/usr/X11R6/lib"
+XLIBPATH="/usr/X11/lib"
 TR=0
 TRINCL=""
 TRLIBS=""
@@ -277,7 +277,7 @@
     CXX="g++"
     CXXFLAG=`echo $OS $OT`
     if [ "$MESADEFAULT" = "1" ] ; then
-	MESAPATH="/usr/X11R6"
+	MESAPATH="/usr/X11"
     fi
     OPENGLINCL="-I${MESAPATH}/include"
     OPENGLLIBS="-L${MESAPATH}/lib -lGL -lGLU -L${XLIBPATH} -lX11 -lXext"
